app/graphql/types/list_type.rb

```diff
@@ -0,0 +1,35 @@
+# frozen_string_literal: true
+
+module Types
+  class ListType < Base::Object
+    I18N_PATH = 'graphql.types.list_type'
+
+    graphql_name 'ListType'
+    implements Types::Interfaces::NodeInterface
+    description I18n.t("#{I18N_PATH}.desc")
+
+    field :name, String, null: false, description: I18n.t("#{I18N_PATH}.fields.name")
+
+    field :items,
+          [Types::MovieType],
+          null: true,
+          description: I18n.t("#{I18N_PATH}.fields.items")
+
+    field :deleted_list_id, ID, null: false, description: I18n.t("#{I18N_PATH}.fields.deleted_list_id")
+
+    def items
+      BatchLoader::GraphQL.for(object.id).batch(default_value: []) do |list_ids, loader|
+        ::Movie
+          .joins(:lists_movies)
+          .where(lists_movies: { list_id: list_ids })
+          .each do |movie|
+            loader.call(movie.list_ids) { |memo| memo << movie }
```

Comment:
does it work correctly? please double check it in insomnia or put binding.pry  it the feature test
if, not you can fetch items like this:
```
BatchLoader::GraphQL.for(object.id).batch(default_value: []) do |list_ids, loader|
        ListsMovie.includes(:movie).where(list_id: list_ids).each do |list_movie|
          loader.call(list_movie.list_id) { |memo| memo << list_movie.movie }
        end
      end
```

Reply:
Yes, it didn't work correctly, I've changed it as you suggested